site/blog_collections/_running/2025-12-31-2025_running_year_in_review.md

```diff
@@ -0,0 +1,56 @@
+
+{% capture table_content %}
+|------------|------------|
+| <img src="/img_running/2025-12-31/strava1.jpg" data-src="/img_running/2025-12-31/strava1.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> | <img src="/img_running/2025-12-31/strava2.jpg" data-src="/img_running/2025-12-31/strava2.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> |
+{% endcapture %}
+{{ table_content | markdownify }}
+
+The volume is slightly less than 2024 (5000K / 501h) - primarily due to Tuesday cross training in the first half of the year (stair stepper, prep for Mozart).
+
+# Miscellaneous Achievements
+
+## Half Marathon Attempts
+This year I chased the sub-90 minute half marathon goal across three races. The results were bittersweet - while my Strava times consistently showed sub-90 performances, the official chip times were all just above the mark. Close, but not quite there! Here are the three attempts:
+
+- [Race: Cross Island Bornholm](/blog/running/2025/04/19/race_cross_island_bornholm) - Official: 1h30m10s, Strava half: 1h29m39s 😆
+- [Race: Odense City Half](/blog/running/2025/05/21/race_odense_city_half) - Official: 1h31m, Strava half: 1h29m46s 😭
+- [Race: Bestseller Aarhus City Half](/blog/running/2025/06/27/race_bestseller_aarhus_city_half) - Official: 1h30m37s, Strava half: 1h29m50s
+
+## ITRA Runner Index
+During the year I reached ITRA Advanced 4 level - a milestone that reflects the accumulation of ultra distance races and vertical gain. The journey to this level has taken me through adventures in Austria (Mozart 100), Sweden (Österlen Spring Trail and Kullamannen), and various trail races in Denmark.
+
+{% capture table_content %}
+|------------|------------|
+| <img src="/img_running/2025-12-31/itra1.jpg" data-src="/img_running/2025-12-31/itra1.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> | <img src="/img_running/2025-12-31/itra2.jpg" data-src="/img_running/2025-12-31/itra2.jpg" alt="" class="spotlight w-100 pl-2 pr-2" style="max-width: 350px" /> |
+{% endcapture %}
+{{ table_content | markdownify }}
+
+## Personal Bests (PBs)
+
+This year brought a number of new personal bests across various distances. Below is a summary of the key achievements organized by distance:
+
+| Distance | Pace | Time | Context |
+|----------|------|------|---------|
+| 100M (160K) | 06:49 /km | 18h11m | Race: Mors 100 Miles |
+| 100K | 06:11 /km | 10h18m29s | Race: LNBK Ringkøbing Fjord Winter Edition |
+| 50K | 05:11 /km | 4h19m20s | Race: LNBK Ringkøbing Fjord Winter Edition |
+| 30K | 04:30 /km | 2h15m | Training |
+| 10K | 03:59 /km | 39m50s | Training |
+| 5K | 03:54 /km | 19m29s | Training: 5K time trial |
+| 1K | 03:38 /km | 3m38s | Race: DHL Stafetten |
+| 1/2 mile | 03:30 /km | 2m49s | Training: Ridaos Intervals |
+| 400m | 03:20 /km | 1m20s | Training: Ridaos Intervals |
```

Comment:
Add comment that I'm really proud of especially the 100 miler in in the 18 hour range. And the 5K in sub 20 minutes. And the 10K in sub 40 minutes (although stars aligned for this to happen: little wind, carbon plate shoes 👌🏻 (Christmas gift 🥳😂) and a fast route with a tiny net decline).